lib/src/page_route.dart

```diff
@@ -20,7 +20,8 @@ class SwipeablePageRoute<T> extends CupertinoPageRoute<T> {
     RouteSettings? settings,
     bool maintainState = true,
     bool fullscreenDialog = false,
-  }) : super(
+    Duration? transitionDuration,
+  }) : _transitionDuration = transitionDuration, super(
```

Comment:
To make the formatter happy:

```suggestion
  })  : _transitionDuration = transitionDuration,
        super(
```